schemas/Quote.json

```diff
@@ -4,6 +4,10 @@
   "description": "A quote object",
   "type": "object",
   "properties": {
+    "source_connector_account": {
+      "description": "the connector's account on the source ledger",
+      "$ref": "Iri.json"
+    },
     "source_ledger": {
```

Comment:
Is it necessary to return the source and destination ledgers (or the amount that you sent in the request)?